services/appflowy-collaborate/src/metrics.rs

```diff
@@ -84,7 +84,7 @@ impl CollabRealtimeMetrics {
 }
 
 #[derive(Clone, Default)]
-pub(crate) struct CollabMetricsCalculate {
+pub struct CollabMetricsCalculate {
```

Comment:
should be private

Reply:
does not compile if it's only public at crate level